middlewares/validators/requests.ts

```diff
@@ -58,3 +60,36 @@ export const updateRequestsMiddleware = async (
     res.boom.badRequest(errorMessages);
   }
 };
+
+export const getRequestsMiddleware = async (req: OooRequestCreateRequest, res: OooRequestResponse, next: NextFunction) => {
+  const schema = joi.object().keys({
+    dev: joi.bool().sensitive(),
+    id: joi.string().optional(),
+    type: joi
+      .string()
+      .valid(REQUEST_TYPE.OOO,"ALL")
```

Comment:
Shouldn't we have all also as a constant?

Reply:
Done in next PR [Commits](https://github.com/Real-Dev-Squad/website-backend/pull/1883/commits/6500f9c0875eaff05ad98455d0323bbaf8b68bac)